lib/graphql/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '3.7'
+
+services:
+  graphql:
+    build:
+      context: $PWD/lib/graphql/GraphQL-interface
+      args:
+        GRAPHQL_CONDA_SOURCE: "${GRAPHQL_CONDA_SOURCE}"
```

Comment:
After review of https://github.com/CanDIG/GraphQL-interface/blob/AliRZ-02/DIG-772-CanDIGv2-Integration/Dockerfile, it seems like Conda is being used unnecessarily. Conda is not needed when deploying in a container. The `scikitlearn` and `pandas` dependencies can be installed on host with `pip`. Additionally, base image in not recommended as it is adds unnecessary overhead.